src/resdk/resolwe.py

```diff
@@ -531,6 +537,22 @@ def __init__(
 
         self.logger.info(f"Successfully logged in as {username}.")
 
+    def version_check(self):
+        """Check that the server is compatible with the client."""
+        url = urljoin(self.url, MINIMAL_SUPPORTED_VERSION_POSTFIX)
+        response = requests.get(url)
+        minimal_version = version.parse(response.data["minimal_supported_version"])
```

Comment:
This currently fails with: `AttributeError: 'Response' object has no attribute 'data'`

If you replace `response.data` with `response.json()` it works as expected.

Reply:
@JureZmrzlikar Thank you for testing. I have changed the code according to your suggestions.